jaxrs-spec/src/main/asciidoc/boilerplate.adoc

```diff
@@ -0,0 +1,45 @@
+﻿//
+//    Copyright (c) 2019 Markus KARG
+//
+//    This program and the accompanying materials are made available under the
+//    terms of the Eclipse Public License v. 2.0, which is available at
+//    http://www.eclipse.org/legal/epl-2.0.
+//
+//    This Source Code may also be made available under the following Secondary
+//    Licenses when the conditions for such availability set forth in the
+//    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+//    version 2 with the GNU Classpath Exception, which is available at
+//    https://www.gnu.org/software/classpath/license.html.
+//
+//    SPDX-License-Identifier: EFSL-1.0
+//
+
+== Boilerplate
+
+WARNING: This document contains just boilerplate information. It explicitly is *not* a complete specification. In particular, it misses all normative text. To learn about compliance with this specification, refer to "JAX-RS: Java™ API for RESTful Web Services" (Version 2.1, Final Release July 13, 2017).
+
+
+=== Scope
+
+*Jakarta RESTful Web Services* provides a specification document, TCK and foundational API to develop web services following the Representational State Transfer (REST) architectural pattern.
+
+
+=== Acronyms
+
+Acronyms used in this document MAY refer to Oracle / JCP technology.
+
+The acronym JAX-RS is reused with permission of Oracle and either refers to "Jakarta RESTful Web Services" or "Java™ API for RESTful Web Services", dependending on the context.
+
+
+=== TCK
+
+NOTE: At time of writing, there is no standalone TCK for *Jakarta RESTful Web Services*. To proof compliance with this specification, pass the Jakarta EE Platform TCK instead.
```

Comment:
I don't believe this is true.  The JAX-RS standalone tck is [here](http://download.eclipse.org/ee4j/jakartaee-tck/8.0.1/nightly/jaxrstck-2.1_latest.zip).

Reply:
Thanks for this link. Can somebody clarify where the source code for this is and why it is not found in the JAX-RS repo?

Reply:
As has been discussed many, many, times, most of the TCKs are produced from the
single [jakartaee-tck](https://github.com/eclipse-ee4j/jakartaee-tck) repository.  Splitting up the TCKs so that they can be moved to
the corresponding spec projects is something that everyone wants, but is a huge job
that will not even be started for Jakarta EE 8.

Reply:
So what you like to tell me is that the Jakarta RESTful Web Service specification shall simply include that link you posted above?

Reply:
The spec doesn't need to link to or reference the TCK at all.  The information about the
TCK will be included in the Release Review for the specification.

Reply:
But what about this requirement we had been given? `The component specifications must also reference the applicable TCK.`?

Reply:
I guess I didn't review that document carefully enough.  :-)

I read that to mean the TCK would be referenced in the Release Review.
Our specs have never referenced the TCK directly and I don't see any reason
to start doing so.  The TCK is part of the Release Review and is required to
claim compatibility.

Reply:
I don't know if we need to include a link or not, but replacing the text above with the link provided by @bshannon  seems more useful than what we have now.